.rubocop.yml

```diff
@@ -17,6 +17,10 @@ Style/FrozenStringLiteralComment:
 Layout/EndOfLine:
   EnforcedStyle: lf
 
+Lint/AmbiguousBlockAssociation:
+  Exclude:
+    - 'spec/**/*_spec.rb'
+
```

Comment:
I'm guessing it's for `expect { ... }.to change { ... }.by

Reply:
Otherwise rubocop flags `expect { sth }.to change { sth }.by 1`, demanding `expect {sth}.to(change {sth}.by(1))` which drives me nuts. :P Happy to discuss though

Reply:
Truth be told I just put the second scope in parenthesis when rubocop shouted at me in E&L tests :D 

That's a good enough reason to disable it there imo.